From 6f57b1631b5ebab03154d2922d02746951665c62 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 3 Jul 2017 15:06:32 +0800
Subject: [PATCH 2071/5242] MLK-15322-10 ARM64: dts: imx8mq-evk: connect mipi
 dsi to adv7535

commit  d20716721c7a6dbb707f1ee39a661ee77e15a6d8 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add port in dsi and adv7535 to connect them.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 616c95d..c23154d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -278,6 +278,11 @@
 		dsi-traffic-mode = <0>;
 		bpp = <24>;
 		status = "okay";
+		port {
+			dsi_to_hdmi: endpoint {
+				remote-endpoint = <&mipi_dsi_ep>;
+			};
+		};
 	};
 };
 
@@ -349,6 +354,11 @@
 	reset = <&src>;
 	mux-sel = <&gpr>;	/* lcdif or dcss */
 	status = "okay";
+	port {
+		mipi_dsi_ep: endpoint {
+			remote-endpoint = <&dsi_to_hdmi>;
+		};
+	};
 };
 
 &uart3 { /* BT */
-- 
1.7.9.5

